app/components/Login/Login.jsx

```diff
@@ -134,7 +134,15 @@ export default class Login extends React.Component {
                 }
             })
             .catch((error) => {
-                this.setState({ errorMessage: `Error: ${error}` });
+                var loginErrorMessage;
+                if (error.response &&
+                    error.response.data &&
+                    error.response.data.errors) {
+                    loginErrorMessage = _.join(error.response.data.errors, ", ");
+                } else {
+                    loginErrorMessage = error.message;
+                }
+                this.setState({ errorMessage: `Error: ${loginErrorMessage}` });
```

Comment:
This a bit verbose. When possible try to use the `lodash` module as it provides a number of helpful functions.

The same code can be written as:
```
if (_.has(error, 'response.data.errors') && error.response.data.errors.length > 0)
    loginErrorMessage = _.join(error.response.data.errors, ", ");
else
    loginErrorMessage = error.message;
```